web-frontend/src/pages/Assets/AssetsList.tsx

```diff
@@ -1,9 +1,21 @@
+import { useState } from 'react';
+
 import { MassaLogo, Mrc20, Token } from '@massalabs/react-ui-kit';
 
+import { DeleteAssetModal } from './DeleteAssetModal';
```

Comment:
why don't you use the @ alias for this one? if this file is moved, you will have to rewrite this import.